src/views/CustomerStory.vue

```diff
@@ -126,9 +126,18 @@ export default {
 
 <style>
```

Comment:
As discussed, please simplify this to:

```html
<style>
.customer-story-logo {
  width: 15rem;
  max-width: 70%;
  display: block;
  margin-left: auto;
  margin-right: auto;
}
</style>
```

Reply:
Thanks! Done. Also introduced a better quality version of the logo.